### PR TITLE
Integrating clib-validate

### PR DESCRIPTION
**Pull request by Isty001**
This PR moves clibs/clib-validate into the main repo, as part of `clib install` and `clib update`

### CODE REVIEW

**Review by jwerle — APPROVED** [review]

File: src/common/clib-validate.c [hunk 1]
```diff
@@ -0,0 +1,93 @@
+
+//
+// clib-validate: `clib-validate.c`
+//
+// Copyright (c) 2021 Stephen Mathieson
```
Comment (stephenmathieson) [inline src/common/clib-validate.c:5]:
```suggestion
// Copyright (c) 2014 Stephen Mathieson
// Copyright (c) 2021 clib authors
```

File: src/common/clib-validate.h [hunk 2]
```diff
@@ -0,0 +1,16 @@
+//
+// clib-release-info.h
+//
+// Copyright (c) 2016-2021 clib authors
```
Comment (stephenmathieson) [inline src/common/clib-validate.h:4]:
```suggestion
// Copyright (c) 2014-2021 clib authors
```

File: src/common/clib-validate.h [hunk 3]
```diff
@@ -0,0 +1,16 @@
+//
+// clib-release-info.h
```
Comment (stephenmathieson) [inline src/common/clib-validate.h:2]:
```suggestion
// clib-validate.h
```

**Review by stephenmathieson — COMMENTED** [review]
Small things, otherwise LGTM 👍

**Comment by Isty001** [comment]
Thanks :+1:  I archived the clibs/clib-validate repo.

**Pull request merged**